awesomplete.js

```diff
@@ -211,14 +216,19 @@ _.prototype = {
 			this.ul.innerHTML = "";
 
 			this.suggestions = this._list
-				.filter(function(item) {
-					return me.filter(item, value);
+				.map(function(item) {
+					var suggestion = new Suggestion(convert(item));
```

Comment:
We only need to wrap with Suggestion here for old string based API inside data method.